lib/bash-utility-master/src/os.sh

```diff
@@ -17,10 +17,11 @@
 #
 # @stdout Operating system name (linux, mac or windows).
 os::detect_os() {
-    declare uname os
+    declare uname os os_lower_case
     uname=$(command -v uname)
-
-    case $("${uname}" | tr '[:upper:]' '[:lower:]') in
+    declare os_lower_case=$("${uname}")
```

Comment:
Results ` os_lower_case: command not found `

Reply:
I did new pull request where problem is solved. There was a typo